File: solutions/0080-remove-duplicates-from-sorted-array-ii/solution.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ls=[]
        for i in set(nums):
            n=nums.count(i)
            if n>=2:
                for j in range(2,n):
                    nums.remove(i)
        return len(nums)

File: solutions/0080-remove-duplicates-from-sorted-array-ii/solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int write=2,n=nums.length;
        for(int i=2;i<n;i++)
            if(nums[i]!=nums[write-2])
                nums[write++]=nums[i];
        return write;
    }
}
